tutor/interactive.py

```diff
@@ -148,6 +157,17 @@ def ask_questions(config: Config, run_for_prod: Optional[bool] = None) -> None:
             config,
             defaults,
         )
+    if clean_environment:
+        run_clean = click.confirm(
+            fmt.question(
+                "The -c flag you provide will delete your existing Tutor environment. "
+                "Are you sure you want to do this?"
+            ),
+            prompt_suffix=" ",
+            default=False,
```

Comment:
I would default True here, just because the user is already going out of their way to provide the `-c` flag.

Reply:
You're absolutely right. changed to True.